brain-marks/Add/AddURLView.swift

```diff
@@ -41,7 +41,7 @@ struct AddURLView: View {
                         viewModel.alertItem = AlertContext.noCategory
                         showingAlert = true
                     } else {
-                        viewModel.fetchTweet(url: newEntry) { result in
+                        viewModel.fetchTweet.async(url: newEntry) { result in
```

Comment:
This issue #141 requires a full refactor of the `fetchTweet` function in the `AddURLViewModel` to change the syntax from using a completion handler to using async / await

It's not possible to add `.async` in this context

See these articles for more information on async / await
* https://www.avanderlee.com/swift/async-await/
* https://www.hackingwithswift.com/swift/5.5/async-await
* https://docs.swift.org/swift-book/LanguageGuide/Concurrency.html